Demo/build.gradle

```diff
@@ -55,6 +55,7 @@ android {
 
 dependencies {
     implementation project(':Card')
+    implementation project(':PayPalCheckout')
```

Comment:
I noticed [here](https://github.com/paypal/iOS-SDK/pull/25/files#r735867417) in iOS we're pulling in native XO via Swift Package Manager. The primary reason we aren't doing the same is because of the 3rd party dependencies?

Reply:
That's definitely the right move I wanted to confirm to give myself proper context on this PR before giving a full review.

Reply:
@sshropshire  Im not pulling native XO here, this is the `build.gradle` of the `Demo` module. On the `build.gradle` of the `PayPalCheckout` module is where im pulling native XO, like this:

`implementation deps.paypalNativeCheckout`

where `deps.paypalNativeCheckout` is `com.paypal.checkout:android-sdk:0.5.1`

Reply:
Oh got it yeah that's my mistake. Did iOS decide to skip the mapping portion? I know we discussed it on Android at one point but I don't remember what they landed on.

Reply:
@sshropshire you mean mapping NXO objects? i posted comment on their PR to see about that. I think that it would be good to at list provide the same interface, otherwise merchant wont be leaving NXO any time soon

Reply:
@jcnoriega Yeah I brought this up in sync. iOS may be pulling in a smaller subset of features to get NXO initially integrated into the project.

Reply:
@sshropshire yeah it seems like it, going to dig in a little bit more to see if that is at least defined somewhere. In the Android ticket its says to include those objects